chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/SchedulerController.java

```diff
@@ -118,7 +124,8 @@ public void deleteTrigger(@PathVariable("triggerGroup") final String triggerGrou
      * @return the updated job
      * @throws SchedulerException if scheduler has issues
      */
-    @ApiOperation(value = "Update a given job")
+    @ApiOperation(value = "Update a given job",
+            notes = "Security Restrictions: ROLE_ADMIN or ROLE_ACB and have administrative authority on the specified ACB.")
```

Comment:
The only job that can be updated is the one that's only available to ROLE_ADMIN and ROLE_ONC, right? I don't think ROLE_ACB has any Jobs they can update

Reply:
Addressed